evm-contracts/src/v0.6-ovm/AccessControlledAggregator.sol

```diff
@@ -1,217 +1,111 @@
 pragma solidity ^0.6.0;
 
-import "./FluxAggregator.sol";
+import "./vendor/Proxy.sol";
+import "./interfaces/AggregatorV2V3Interface.sol";
 import "./SimpleReadAccessController.sol";
 
 /**
  * @title AccessControlled FluxAggregator contract
  * @notice This contract requires addresses to be added to a controller
  * in order to read the answers stored in the FluxAggregator contract
  */
-contract AccessControlledAggregator is FluxAggregator, SimpleReadAccessController {
+contract AccessControlledAggregator is Proxy, SimpleReadAccessController {
```

Comment:
I would agree, but this simple change could introduce substantial changes to tooling, Gauntlet as an example. I'll investigate how much. 

This is an issue only because of the time crunch we are in.